src/sage/graphs/orientations.py

```diff
@@ -171,7 +178,6 @@ def strong_orientations_iterator(G):
         # launch the algorithm for enumeration of the solutions
         for sol in _strong_orientations_of_a_mixed_graph(Dg, V, treeEdges):
```

Comment:
this can be replaced by `yield from ....`

Reply:
Indeed.